integration_tests/pages/overview/EducationAndTrainingPage.ts

```diff
@@ -74,6 +74,11 @@ export default class EducationAndTrainingPage extends Page {
     return Page.verifyOnPage(InPrisonWorkPage)
   }
 
+  clickViewAllCoursesAndQualificationsLink(): EducationAndTrainingPage {
```

Comment:
I think this method should be returning a `InPrisonCoursesAndQualificationsPage`, which might be why the tests are failing at the moment?
```
clickViewAllCoursesAndQualificationsLink(): InPrisonCoursesAndQualificationsPage {
    this.viewAllCoursesAndQualificationsLink().click()
    return Page.verifyOnPage(InPrisonCoursesAndQualificationsPage)
  }
```

Reply:
I hadn't enabled the new feature toggle in the `feature.env`!

Reply:
No, this method is wrong. It needs to be as I wrote above.
If you've clicked a link that takes you to a new page, then the return type of the method should be the target page